airbrakes/state.py

```diff
@@ -150,21 +149,11 @@ def update(self):
         # In Coast State we start predicting the apogee
         self.context.predict_apogee()
 
-        # Check if we are going to overshoot our target apogee, and extend the airbrakes if we are.
-        pred_apogee = self.context.apogee_predictor.apogee
-
-        if pred_apogee is not None and pred_apogee >= TARGET_ALTITUDE:
-            self.context.extend_airbrakes()
-
         data = self.context.data_processor
 
-        # if our velocity is close to zero or negative, we are in free fall.
-        if data.vertical_velocity <= 0:
-            self.next_state()
-            return
-
-        # fallback condition:
-        # if our altitude has started to decrease, we have reached apogee:
```

Comment:
should've been commented out at least..

Reply:
no but its just wrong and not how we will do it

Reply:
Oh if you're talking about the vertical velocity thing, I just don't see any benefit of having it. Only a potential downside.